<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Test_Sim</class>
 <widget class="QMainWindow" name="Test_Sim">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="4">
     <widget class="QPushButton" name="button">
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="Led" name="led">
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QCheckBox" name="checkBox">
      <property name="text">
       <string>CheckBox</string>
      </property>
     </widget>
    </item>
    <item row="4" column="4">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="dial_label">
        <property name="text">
         <string>0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QwtDial" name="dial">
        <property name="lineWidth">
         <number>4</number>
        </property>
        <property name="minScaleArc">
         <double>20.000000000000000</double>
        </property>
        <property name="maxScaleArc">
         <double>340.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="4">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="slider_label">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QwtSlider" name="slider"/>
      </item>
     </layout>
    </item>
    <item row="4" column="2">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="thermo_label">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QwtThermo" name="thermo"/>
      </item>
     </layout>
    </item>
    <item row="3" column="4">
     <widget class="QComboBox" name="combobox"/>
    </item>
    <item row="5" column="1">
     <widget class="Plot_wrapper" name="plot">
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QTableWidget" name="table"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtDial</class>
   <extends>QWidget</extends>
   <header>qwt_dial.h</header>
  </customwidget>
  <customwidget>
   <class>QwtSlider</class>
   <extends>QWidget</extends>
   <header>qwt_slider.h</header>
  </customwidget>
  <customwidget>
   <class>QwtThermo</class>
   <extends>QWidget</extends>
   <header>qwt_thermo.h</header>
  </customwidget>
  <customwidget>
   <class>Led</class>
   <extends>QPushButton</extends>
   <header>led.hpp</header>
  </customwidget>
  <customwidget>
   <class>Plot_wrapper</class>
   <extends>QFrame</extends>
   <header>plot_wrapper.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
